database/conferences.json

```diff
@@ -309,5 +309,21 @@
         "publication_type": "conferencepaper",
         "access_right": "open",
         "license": "CC-BY-4.0"
+    },
+    "2019": {
```

Comment:
We don't have the program committee listed in this file, it means that there's no machine readable location place where we can get that information. Specifically, it means that generating the dblp file will require manual editing to add that data. Do we want to consider adding PC here?

Reply:
Good catch. Let's add them. Start with 2019 as an array an we add the previous ones later. --> Issue

Reply:
Yes, let's get them in. Add here first and we'll add the previous ones later.